File: app/src/main/java/com/silentgames/silent_planet/mvp/main/MainViewModel.java
package com.silentgames.silent_planet.mvp.main;

import com.silentgames.silent_planet.model.Cell;
import com.silentgames.silent_planet.model.GameMatrixHelper;

import java.util.Map;

/**
 * Created by gidroshvandel on 21.06.17.
 */
public class MainViewModel {

    private GameMatrixHelper gameMatrixHelper;

    private boolean doubleClick;

    public GameMatrixHelper getGameMatrixHelper() {
        return gameMatrixHelper;
    }

    public void setGameMatrixHelper(GameMatrixHelper gameMatrixHelper) {
        this.gameMatrixHelper = gameMatrixHelper;
    }

    public boolean isDoubleClick() {
        return doubleClick;
    }

    public void setDoubleClick(boolean doubleClick) {
        this.doubleClick = doubleClick;
    }
}

File: app/src/main/java/com/silentgames/silent_planet/mvp/main/MainViewModel.kt
package com.silentgames.silent_planet.mvp.main

import com.silentgames.silent_planet.logic.ecs.EngineEcs
import com.silentgames.silent_planet.logic.ecs.entity.unit.UnitEcs

/**
 * Created by gidroshvandel on 21.06.17.
 */
class MainViewModel {

    lateinit var engine: EngineEcs

    var selectedEntity: UnitEcs? = null

}
